### PR TITLE
Documented the clamp function

### PR DESCRIPTION
**Pull request by octoflo**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
-"""Standard utilies"""
+"""
+Standard utilies
```
Comment (calebeby) [inline utils.py:2]: At the top of the file is where you describe what kinds of things belong in this file, if it is not immediately obvious. I think `Standard utilities` is sufficient to describe the kinds of things that will be in this file.

Documentation for specific functions does not belong at the top of the file; instead, it belongs inside each function, at the top of each function.

File: utils.py [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+You can use this function if you have an input value and want to make sure it's in a specified range. 
+For example, if you have a joystick value that's 1.1 and you called the clamp function with a min of 0 and a max of 1
+it will change the joystick value from 1.1 to 1.
+"""
 
-
-def clamp(value, max, min):
-    """Clamp value within a range"""
-    return max if value > max else min if value < min else value
+def clamp(value, max, min): # this function takes the input value and your specified range
```
Comment (calebeby) [inline utils.py:8]: It is unnecesary to repeat the parameters in a comment.

File: utils.py [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+"""
+Standard utilies
+You can use this function if you have an input value and want to make sure it's in a specified range. 
+For example, if you have a joystick value that's 1.1 and you called the clamp function with a min of 0 and a max of 1
+it will change the joystick value from 1.1 to 1.
+"""
 
-
-def clamp(value, max, min):
-    """Clamp value within a range"""
```
Comment (calebeby) [inline utils.py:5]: Why did you remove the function docstring?

**Review by calebeby — CHANGES_REQUESTED** [review]

**Comment by brendanburkhart** [comment]
Closed due to inactivity

**Pull request closed without merging**